pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerializeJavaR.scala

```diff
@@ -0,0 +1,297 @@
+package edu.berkeley.cs.amplab.sparkr
+
+import scala.collection.JavaConversions._
+
+import java.io.DataInputStream
+import java.io.DataOutputStream
+
+/**
+ * Utility functions to serialize, deserialize objects to / from R
+ */
+object SerializeJavaR {
```

Comment:
Or you can even just `object serde { ... }` -- it's not a class so I think it's fine to lower-case the first letter